rpc/clients/typescript/src/types.ts

```diff
@@ -1,5 +1,5 @@
 import { SignedOrder } from '@0x/types';
-import { BigNumber } from '@0x/utils';
```

Comment:
Is it okay to import `bignumber.js` directly here? IIRC in the past we were configuring `bignumber.js` a certain way in `@0x/utils` so we needed to always import from there.

Reply:
That is a really good point. Thanks for catching this. I've gone ahead and temporarily added the same BigNumber configs that are in `@0x/utils` to our client.